ton_sdk/src/contract.rs

```diff
@@ -14,15 +14,16 @@
 
 use crate::*;
 use ed25519_dalek::{Keypair, PublicKey};
-use std::convert::Into;
+use chrono::prelude::*;
```

Comment:
It is bad practice to use `*` import

Reply:
It is code from example in crate documentation.
https://crates.io/crates/chrono